release.yaml

```diff
@@ -60,3 +60,7 @@ sriov-crd:
 rancher-webhook:
 - 1.0.8+up0.2.9
 - 0.1.600+up0.1.6
```

Comment:
I believe the previously released charts will need to be removed from the `release.yaml` for an OOB release.

Reply:
Resolved